server/fly.io/src/main.rs

```diff
@@ -0,0 +1,76 @@
+use std::collections::BTreeMap;
+use std::net::SocketAddr;
+
+use axum::{
+    extract::Query,
+    routing::{get, post},
+    Router,
+};
+use influxrs::{InfluxClient, Measurement};
+
+#[tokio::main]
+async fn main() {
+    let app = Router::new()
+        .route("/", get(root))
+        .route("/record-install", get(record_install))
+        .route("/record-install", post(record_install));
+
+    // ipv6 + ipv6 any addr
+    let addr = SocketAddr::from(([0, 0, 0, 0, 0, 0, 0, 0], 8080));
+    axum::Server::bind(&addr)
+        .serve(app.into_make_service())
+        .await
+        .unwrap();
+}
+
+async fn root() -> &'static str {
+    "Hello, World!"
+}
+
+fn get_env(key: &str) -> String {
+    std::env::var(key).expect(&format!("{key} must be set"))
+}
+
+async fn record_install(Query(params): Query<BTreeMap<String, String>>) -> String {
+    println!("Hi there {params:?}");
+
+    // FIXME: make this in main and pass it down or something?
+    let url = get_env("INFLUX_URL");
+    let token = get_env("INFLUX_TOKEN");
+    let org = get_env("INFLUX_ORG");
+    let bucket = get_env("INFLUX_BUCKET");
+    let client = InfluxClient::builder(url, token, org).build().unwrap();
+
+    let mut point = Measurement::builder("counts").field("count", 1);
+    for (tag, value) in &params {
+        point = point.tag(tag, &**value)
```

Comment:
I had a go at playing with the data:
<img width="1245" alt="Screenshot 2023-03-05 at 13 06 31" src="https://user-images.githubusercontent.com/254647/222962406-fe395f21-03a6-487f-96f6-e8ac5fd87b1f.png">

-- https://eu-central-1-1.aws.cloud2.influxdata.com/orgs/69235d4f38c3e042/notebooks/61ac6f8ba31b7000

Feel free to mess around with it and see what you find.

Reply:
backlogged shared grafana dashboards as https://github.com/cargo-bins/cargo-quickinstall/issues/279